test/cpp/test_xla_sharding.cpp

```diff
@@ -33,7 +33,13 @@ bool XlaDataValuesEqual(torch::lazy::BackendDataPtr a,
 }
 }  // namespace
 
-class XLAShardingTest : public AtenXlaTensorTestBase {};
+class XLAShardingTest : public AtenXlaTensorTestBase {
+ protected:
+  static void SetUpTestCase() {
+    setenv("XLA_USE_SPMD", "1", /*overwrite=*/true);
```

Comment:
big picture question: what's our plan to remove `XLA_USE_SPMD` flag to improve usability?

@yeounoh @vanbasten23

Reply:
iiuc, on the user facing level (python), we want to replace `XLA_USE_SPMD` with an API use_spmd(). Internally, it seems fine to keep using the flag since it doesn't affect usability. Here we use the flag only internally.